lib/renderful.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'renderful/error/base'
+require 'renderful/error/entry_not_found_error.rb'
```

Comment:
You can omit `.rb` when requiring files.

Reply:
👍 paste error 😅